test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
@tomalec @warpech this just bit me. When you `bower install` in Starcounter ClientFiles. `puppet-client` depends on `Palindrom#`**`master`**. And Palindrom master uses `JSONPatcherProxies`. But `JSONPatcherProxies` isn't installed and an 404 error occurs.

I guess bower deps are needed? Does Starcounter install using NPM?

As far as I understand, we either make `puppet-client` use Palindrom bundle and release a patch ASAP, or we put bower deps back in Palindrom's master. 

Please correct me if I am wrong, I have thought-spaghetti.